cos-fleetshard-operator-camel/src/main/java/org/bf2/cos/fleetshard/operator/camel/model/KameletBinding.java

```diff
@@ -42,6 +42,7 @@
         .build();
 
     public KameletBinding() {
+        super();
```

Comment:
@lburgazzoli  I tried adding this. But still my local tests fail with the `KameletBinding` resource is completely empty.

Reply:
which test ?

Reply:
@lburgazzoli  https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/runs/5192522455?check_suite_focus=true#step:6:1559

Reply:
Could this still be related to #208  / MGDCTRS-458?

Reply:
It should not as the failing test is a unit test that does not have interactions with kubernetes

Reply:
the issue is related to the fact that `CamelOperandControllerTest` now also have data shapes but the relates assertions have not been amended accordingly. 

I'll send a pr to your branch

Reply:
PR sent https://github.com/bf2fc6cc711aee1a0c2a/cos-fleetshard/pull/213

Reply:
Thx for the PR @lburgazzoli. I will check if it resolves the issues and push the update to not make `data_shape` mandatory now aka removing the exception. But mind! :warning: some tests might fail as sometimes `data_shape` properties (the previous JsonNode) was in some cases expected to be `null` and in other cases to be `MissingNode`. Some Camel operator logic might be broken, but I'll do my best to harmonize it with this PR.

Reply:
If you ruin into blockers, let me know and I'll try to fix it

Reply:
it works now! thx a lot @lburgazzoli :rocket: